syntax = "proto3";
option go_package = "./grpc/";

service ClassificateImage {
  rpc Classificate (ImageClassificationRequest) returns (ImageClassificationResponse) {}
}
message ImageClassificationRequest {
  string extenstion = 1;
  int32 height = 2;
  int32 width = 3;
  bytes image = 4;
}

message ImageClassificationResponse {
  string DiseaseName = 1;
}
